### PR TITLE
docs: second set of video contents added

### PR DESCRIPTION
**Pull request by Kikiodazie**
This PR adds the second set of video content (eCHO episodes and some talks) to the docs.

### CODE REVIEW

File: Documentation/gettingstarted/local-redirect-policy.rst [hunk 1]
```diff
@@ -16,6 +16,11 @@ or Kubernetes service to be redirected locally to backend pod(s) within a node,
 using eBPF. The namespace of backend pod(s) need to match with that of the policy.
 The CiliumLocalRedirectPolicy is configured as a ``CustomResourceDefinition``.
 
+.. admonition:: Video
+  :class: attention
+
+  Aside from this document, you can watch a video explanation of Cilium's Local Redirect Policy on `eCHO episode 39 <https://www.youtube.com/watch?v=BT_gdlhjiQc&t=176s>`__.
```
Comment (qmonnet) [inline Documentation/gettingstarted/local-redirect-policy.rst]: Missing episode title?

File: Documentation/gettingstarted/host-firewall.rst [hunk 2]
```diff
@@ -13,6 +13,11 @@ Host Firewall
 This document serves as an introduction to Cilium's host firewall, to enforce
 security policies for Kubernetes nodes.
 
+.. admonition:: Video
+  :class: attention
+  
+  You can also watch a video of the Cilium's host firewall in action on `eCHO Episode 40: Cilium Host Firewall <https://www.youtube.com/watch?v=GLLLcz398K0&t=288s>`__.
```
Comment (qmonnet) [inline Documentation/gettingstarted/host-firewall.rst]:
```suggestion
  You can also watch a video of Cilium's host firewall in action on
  `eCHO Episode 40: Cilium Host Firewall <https://www.youtube.com/watch?v=GLLLcz398K0&t=288s>`__.
```

File: Documentation/gettingstarted/cni-chaining-aws-cni.rst [hunk 3]
```diff
@@ -20,6 +20,13 @@ and provide encryption.
 
 .. include:: cni-chaining-limitations.rst
 
+.. admonition:: Video
+  :class: attention
+
+  If you require some advanced features of Cilium, consider migrating fully to Cilium. 
+  To help you migrate, you can watch two Principal Engineers at Meltwater talk about `how they migrated
```
Comment (qmonnet) [inline Documentation/gettingstarted/cni-chaining-aws-cni.rst]: Repetitions of “Cilium” and “migrat*”. Could be at least “To help you with the process”.

Also “some advanced features” -> just “advanced features”? The word “some” doesn't bring anything here.

File: Documentation/gettingstarted/bandwidth-manager.rst [hunk 4]
```diff
@@ -242,3 +242,8 @@ Limitations
     * Bandwidth enforcement doesn't work with nested network namespace environments like Kind. This is because
       they typically don't have access to the global sysctl under ``/proc/sys/net/core`` and the
       bandwidth enforcement depends on them.
+
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's bandwidth manager, check out this `KubeCon talk on Better Bandwidth Management with eBPF <https://youtu.be/QTSS6ktK8hY>`__.
```
Comment (qmonnet) [inline Documentation/gettingstarted/bandwidth-manager.rst]: youtube.com URL, for consistency with the other additions

File: FURTHER_READINGS.rst [hunk 5]
```diff
@@ -17,8 +17,28 @@ Related Material
 Presentations
 -------------
 
+* Kubernetes on Edge Day, Europe 2022 - Connecting Klusters on the Edge with Deep Dive into Cilium Cluster Mesh:
+  `Video <https://youtu.be/UcsEVnFtrLY>`__
```
Comment (qmonnet) [inline FURTHER_READINGS.rst]: youtube.com URLs please, for consistency with the other links below.

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks! Some fixes and suggestions.

Can you please fold the spell list updates into the relevant commits?

File: Documentation/gettingstarted/local-redirect-policy.rst (hunk 1 above)
Comment (Kikiodazie) [reply Documentation/gettingstarted/local-redirect-policy.rst]: I have made the edit. Do check it out.

File: Documentation/gettingstarted/host-firewall.rst (hunk 2 above)
Comment (Kikiodazie) [reply Documentation/gettingstarted/host-firewall.rst]: Applied. Thank you.

File: Documentation/gettingstarted/cni-chaining-aws-cni.rst (hunk 3 above)
Comment (Kikiodazie) [reply Documentation/gettingstarted/cni-chaining-aws-cni.rst]: Made an edit. Please to check it out.

File: Documentation/gettingstarted/bandwidth-manager.rst (hunk 4 above)
Comment (Kikiodazie) [reply Documentation/gettingstarted/bandwidth-manager.rst]: Made the edit. Thank you.

File: FURTHER_READINGS.rst (hunk 5 above)
Comment (Kikiodazie) [reply FURTHER_READINGS.rst]: Fixed all.

File: Documentation/gettingstarted/cni-chaining-aws-cni.rst [hunk 6]
```diff
@@ -20,6 +20,13 @@ and provide encryption.
 
 .. include:: cni-chaining-limitations.rst
 
+.. admonition:: Video
+  :class: attention
+
+  If you require advanced features of Cilium, consider migrating fully to Cilium. 
+  To help you with the process, you can watch two Principal Engineers at Meltwater talk about `how they migrated
```
Comment (qmonnet) [inline Documentation/gettingstarted/cni-chaining-aws-cni.rst]: trailing spaces at the end lines

**Review by qmonnet — CHANGES_REQUESTED** [review]
Nearly all good, thanks for the changes!

Can you please fold the spell list updates into the relevant commits, as suggested in the previous review?

File: Documentation/gettingstarted/cni-chaining-aws-cni.rst (hunk 6 above)
Comment (Kikiodazie) [reply Documentation/gettingstarted/cni-chaining-aws-cni.rst]: Fixed. Thank you.

**Comment by Kikiodazie** [comment]
> Nearly all good, thanks for the changes!
> 
> Can you please fold the spell list updates into the relevant commits, as suggested in the previous review?

Ohh. So sorry I missed the spell list comment earlier. Fixed. Please do check it out.

**Review by christarazi — COMMENTED** [review]
The changes are good, thanks for the PR!

I am wondering though: do we really need 6 commits to add links to the docs? I think it makes sense to squash these all together.

**Review by qmonnet — APPROVED** [review]
Looks all good, thanks a lot!

> I am wondering though: do we really need 6 commits to add links to the docs? I think it makes sense to squash these all together.

Could have been one commit, but Divine asked beforehand (to Bill, Liz and myself I think, during a GSoD sync-up) if one commit per link was fine and we agreed on that. Let's keep this for this PR at least, unless you strongly object @christarazi.

**Review by joestringer — APPROVED** [review]

**Comment by joestringer** [comment]
Split commits is fine, it might even make it easier to rebase now that there's a conflict.

**Comment by christarazi** [comment]
@qmonnet Fine with me :heavy_check_mark:

**Review by christarazi — APPROVED** [review]

**Comment by qmonnet** [comment]
@Kikiodazie Can you please rebase this PR to address the conflicts?

**Comment by Kikiodazie** [comment]
> @Kikiodazie Can you please rebase this PR to address the conflicts?

Done @qmonnet. So sorry for getting to this late. Was Away from my keyboard all day.

**Comment by borkmann** [comment]
@Kikiodazie Thanks! Could you also add https://youtu.be/OIyPm6K4ooY to the kube-proxy free guide?

**Comment by Kikiodazie** [comment]
> @Kikiodazie Thanks! Could you also add https://youtu.be/OIyPm6K4ooY to the kube-proxy free guide?

Thank you @borkmann. I have already added this video in the docs via this commit https://github.com/cilium/cilium/pull/20480/commits/a157dfebdefc97b6c9631b3abdb9a5dd5d535570, but i can also add it to the to the kube-proxy free guide. 

Please where do you think its best I add it in the kube-proxy free guide?

**Pull request merged**